scripts/building/check_enforcer.py

```diff
@@ -0,0 +1,288 @@
+# Import necessary libraries
```

Comment:
add some docstring on what this tool does

Reply:
Okay